### DISCUSSION TITLE
Kiai fountains don't enter/appear gradually if previous ones are still visible

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#26622] [Q&A] [answered]
(The title is only an inference. In the example below, several kiai effects occur in short intervals.)

The issue can be reproduced well with [this beatmap](https://osu.ppy.sh/beatmapsets/1388906).

This doesn't look intended.
However, I must admit that with the beatmap example I gave, the fountain looks cooler this way! (check with the music.)
Not sure if you can benefit by keeping this as intended.

Here's how it looks in:
- stable
  
  https://github.com/ppy/osu/assets/43073074/8a04502f-2756-4ee3-8034-195f5bf33f68
- lazer
  
  https://github.com/ppy/osu/assets/43073074/dae49f28-8852-4842-bce5-6bc45474c0c6

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26667
